modules/combined/test/tests/reference_residual/tests

```diff
@@ -25,4 +25,31 @@
                   Variables/temp/scaling=0.001'
       prereq = test_sm
     [../]
+    [./group_variables]
+      type = 'Exodiff'
+      input = 'group_variables.i'
+      exodiff = 'group_variables_out.e'
+      abs_zero = 1e-9
+      requirement = 'For ReferenceResidualProblem, groups two standard variables and two scalar variables, respectively.'
```

Comment:
A little rewording on these requirements:

The reference residual based convergence checking system shall permit the norm of independent groups of either scalar and standard solution variables to be compared against reference quantities provided through the norms of groups of auxiliary variables.